invisible_cities/core/fit_functions_test.py

```diff
@@ -289,6 +289,18 @@ def test_fit_with_errors(reduced):
     assert_allclose(f.values, pars)
 
 
+def test_fixed_parameters():
+    pars = [3.0,  2.0, 0.50]
+    x = np.arange(10.)
+    y = fitf.gauss(x, *pars)
+    e = 0.1 * y
+
+    fixed_mu = fitf.fixed_parameters(fitf.gauss, mu = pars[1])
+    seeds = np.array(pars[::2])
+    f = fitf.fit(fixed_mu, x, y, seeds * 1.2, sigma=e)
+    assert_allclose(f.values, seeds)
+
+
```

Comment:
Can I ask for a couple of simple tests that check that it explodes in the appropriate cases? Namely:
- Try to fix all parameters and check that raises an exception (the exact one you expect)
- Try to keep all parameters free, it should also raise an exception
- Try to send unknown parameters, it should raise something as well

Also, you can add a test previous to `test_fixed_parameters` that does some basic checks like ensuring that the number of parameters in the returned function is correct for a number of cases (`pytest.mark.parametrize` is useful here).